### ISSUE TITLE
Quest hotkey listed in Paper in starting inventory does not match config

### ISSUE THREAD

**Issue by lily-es** [Divine-Journey-2/main#163]
**Modpack version:**
2.0.0

**Description:**
The paper in the inventory says to press "K" to open quests, but the hotkey to open the quests corresponds to the grave key insted. Pressing K enables rotation lock.

**Steps to reproduce:**
1. Start new world
2. Read paper item
3. Press K
4. Rotation locks activated

**Expected behavior:**
Quests window should open upon pressing K.

**Comment by RammyT** [comment]
Could it be that you didn't grab the options.txt when you downloaded the pack?

**Comment by lily-es** [comment]
I followed the instructions written in the discord. Extracted the zip into the instance folder. Was there any extra files I was supposed to grab?

**Comment by RammyT** [comment]
No, the options.txt should've been in there so that's not it then.

**Comment by Atricos** [comment]
Does anyone else have this issue? I tested it and the default keybind *is* K for me.

Question for @lily-es: I want to know if the other configs are working correctly. An easy way to check this is to see if "Auto Jump" is disabled by default or not.

**Comment by lily-es** [comment]
Its possible it has to do with the way it was installed. Maybe with a
properly formatted zip and importing the instance from the zip, the config
located in /config/defaultoptions would then be copied over to the main
instance folder, but in my case I simply created a custom 1.12.2 instance
with MultiMC and extracted the zip from github onto it. I did this both
before starting up the pack and afterwards just to be safe.

**Comment by Atricos** [comment]
Okay, try this: Delete your options.txt (located in your main modpack folder), and relaunch the game.

**Comment by KleinGrrmpf** [comment]
It works for me too. K is my default keybind for the quests

**Comment by lily-es** [comment]
Created a new instance from curseforge and the issue is still present.

**Comment by Atricos** [comment]
I created a new CurseForge instance from the Divine Journey 2.0.1 download, loaded up, made a new world, and "K" works to open my quests.

What is your "Open Quests" button bound to?
![image](https://user-images.githubusercontent.com/24854267/131350050-1311856a-3cfb-4aeb-b680-ff7d01a340e3.png)

**Comment by lily-es** [comment]
It was bound to the grave key

**Comment by lily-es** [comment]
After doing some research, it seems like its possible that Optifine might be breaking the DefaultOptions mod

**Comment by lily-es** [comment]
Either way 1.12 is not supported anymore by the mod creator so im not expecting a fix.
